templates/web/base/govuk/fields.html

```diff
@@ -334,3 +336,9 @@ <h2 class="govuk-fieldset__heading">
   <input class="govuk-file-upload" id="[% field.id %]" name="[% field.id %]" type="file">
   </div>
 [% END %]
+
+[% BLOCK compound %]
```

Comment:
I may not actually need this change given how I render the form elements individually now.